transformer_engine/common/fused_attn/fused_attn_f16_arbitrary_seqlen.cu

```diff
@@ -311,10 +313,14 @@ void fused_attn_arbitrary_seqlen_fwd_impl(
           offset_v, offset_o, dropout_seed, dropout_offset] =
         get_graph(sdpa_f16_fprop_cache, descriptor);
 
-    auto plan_workspace_size = mha_graph->get_workspace_size();
     // Exit to request upper level API to allocate memory if needed
-    size_t actual_seqlen_workspace_size = 2 * b * sizeof(int32_t);
-    size_t seqlen_offsets_workspace_size = 4 * (b + 1) * sizeof(int32_t);
+    // n.b. Care should be taken to align each of the added worksapce tensors to their type.
```

Comment:
Is the alignment value based on the type, or is it fixed for all types because of some cuDNN implementation requirements?

"worksapce -> workspace" :)

Reply:
Alignment was chosen to be 16 since that is the largest alignment noted in https://docs.nvidia.com/cuda/cuda-c-programming-guide/#built-in-vector-types Its not a requirement but seemed like a natural value to use.

Reply:
I think actually 8 bytes alignment is enough for int64_t, but 16 bytes is also fine.